src/Type/MixedType.php

```diff
@@ -146,7 +146,7 @@ public function unsetOffset(Type $offsetType): Type
 
 	public function flipArray(): Type
 	{
-		return new self($this->isExplicitMixed);
+		return new ArrayType(new MixedType(), new MixedType());
```

Comment:
I think the key and value should keep the explicitness of this MixedType value.

Reply:
good point, yeah, let's try that 😊
adapted